Gemfile

```diff
@@ -2,19 +2,6 @@ source 'https://rubygems.org'
 
 gemspec
 
-# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
-gem 'gettext', '>= 3.1.3', '< 4.0.0'
-gem 'rake', '~> 13.0'
-
-group :test do
```

Comment:
In general I think test groups in Gemfile is a good pattern

Reply:
I thought it was necessary to include them in the gemspec file to ensure proper handling of dependencies during gem installation for GHA workflows, similar to what was done in this [PR](https://github.com/Katello/hammer-cli-katello/pull/924).

Reply:
That one was a bit special in that there were overlapping dependencies. Specifying something twice is a bad practice. I would have liked to move them to Gemfile instead, but there it was more about deduplicating. In some places we don't install the development dependencies in testing. Then the Gemfile is more flexible because it allows multiplle groups